src/confd/src/infix-if-gre.c

```diff
@@ -5,17 +5,18 @@
 
 int gre_gen(struct lyd_node *dif, struct lyd_node *cif, FILE *ip)
 {
-	const char *ifname, *local, *remote;
+	const char *local, *remote;
 	struct lyd_node *gre;
 	int ipv6;
 
-	ifname = lydx_get_cattr(cif, "name");
 	gre = lydx_get_child(cif, "gre");
-	local = lydx_get_cattr(gre, "local");
+	local = lydx_get_cattr(gre, "local"),
```

Comment:
Even if this is a valid construct, i think we should avoid it.

Reply:
Ouch! Good catch, that was med copy-pasting things back and forth :sweat_smile: